### PR TITLE
✨ Format timestamps in tracker table

### PR DESCRIPTION
**Pull request by superbuggy**
# [OSIDB-2983] [OSIM: Properly format tracker timestamps]

## Checklist:

- [x] Linting passed
- [x] Type checks passed
- [x] Tests suite passed
- [x] Commits consolidated
- [x] Changelog updated
- ~~[ ] Test cases added/updated~~
- [x] Jira ticket updated

## Summary:

[Replace with a brief summary of the changes introduced by this PR.]

## Changes:

[Replace with a detailed description of the changes made in this PR, including any new features, enhancements, bug fixes, or refactorings.]

## Considerations:

[Replace with any additional considerations, notes, or instructions for reviewers.]

### CODE REVIEW

**Review by d10n — APPROVED** [review]

**Review by tboyetteredhat — APPROVED** [review]

File: src/utils/helpers.ts [hunk 1]
```diff
@@ -92,3 +93,8 @@ export const deepMap = (transform: (arg: any) => any, object: DeepMappable): any
 
 export const cveRegex = /^CVE-(?:1999|2\d{3})-(?!0{4})(?:0\d{3}|[1-9]\d{3,})$/;
 export const uniques = <T>(array: T[]) => Array.from(new Set(array));
+
+export function formatDate(date: Date | string): string {
+  const jsDate = new Date(date); // Handles strings in ISO/component format, and Date object
+  return DateTime.fromJSDate(jsDate, { zone: 'utc' }).toFormat('yyyy-MM-dd T ZZZZ');
+}
```
Comment (MrMarble) [inline src/utils/helpers.ts:100]: We have almost this exact function in `EditableDate` widget, maybe remove it from the widget and use this helper https://github.com/RedHatProductSecurity/osim/blob/9f06e2dd03988f1236bca8288cfa1ff04e1d01c2/src/components/widgets/EditableDate.vue#L80-L83

Comment (C-Valen) [reply src/utils/helpers.ts:100]: +1, if @superbuggy doesn't mind to update it.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,11 @@
 # OSIM Changelog
 
+## [2024.6.2]
+### Added
+
+### Fixed
+* Formatting for Tracker table timestamps
```
Comment (C-Valen) [inline CHANGELOG.md:7]: Missed OSIDB ticket?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,11 @@
 # OSIM Changelog
 
+## [2024.6.2]
```
Comment (C-Valen) [inline CHANGELOG.md:3]: Doesn't have sense to use `unreleased` until we have clear the new release to be made, so we will get the specific items we need from unreleased and move it to the new release block?

**Review by C-Valen — CHANGES_REQUESTED** [review]
Not really requesting changes but would like to clarify marked points.

Implementation looks good, and nice to have a reusable date formatting on helper

File: src/utils/helpers.ts (hunk 1 above)
Comment (superbuggy) [reply src/utils/helpers.ts]: I refactored the helper function so that it's reuseable in `EditableDate`. It seems to be working as imtended.

**Review by C-Valen — APPROVED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (superbuggy) [reply CHANGELOG.md]: Yes--I think I misunderstood the last team conversation about release documenting, thinking what we were doing for a particular release was an overall change in process to move away from using an unreleased heading.

**Comment by superbuggy** [comment]
@MrMarble @d10n would you mind looking this once more? I changed the logic in `EditableDate`

**Review by MrMarble — APPROVED** [review]

**Pull request merged**